openbas-front/tests_e2e/fixtures/baseFixtures.ts

```diff
@@ -1,28 +1,52 @@
-/* eslint-disable  @typescript-eslint/no-explicit-any, no-underscore-dangle */
-import { test as baseTest } from '@playwright/test';
-import * as crypto from 'crypto';
-import * as fs from 'fs';
-import * as path from 'path';
+import { Page, test as testBase } from '@playwright/test';
+import MCR from 'monocart-coverage-reports';
 
-const istanbulCLIOutput = path.join(process.cwd(), '.nyc_output');
+import coverageOptions from '../conf/mcr.config';
 
-export function generateUUID(): string {
-  return crypto.randomBytes(16).toString('hex');
-}
+// fixtures
+const test = testBase.extend<{
+  autoTestFixture: string;
+}>({
+  autoTestFixture: [async ({ context }, use) => {
+    const activateCoverage = process.env.E2E_COVERAGE;
+    const isChromium = test.info().project.name === 'chromium';
```

Comment:
Suggestion: comment in `playwright.config.ts` to advise against changing the project's name, as this is now hardcoded in the test code.